pom.xml

```diff
@@ -42,9 +42,7 @@
     </developers>
 
     <properties>
-        <jaxb-api.version>2.4.0-b180830.0359</jaxb-api.version>
-        <jaxb-runtime.version>2.4.0-b180830.0438</jaxb-runtime.version>
-        <jib.from.image>gridsuite/java-simulator:2.1.0</jib.from.image>
+        <jib.from.image>gridsuite/java-dynawo</jib.from.image>
```

Comment:
this image uses java11 (https://hub.docker.com/layers/gridsuite/java-dynawo/latest/images/sha256-b143836b6245068ab039e9543a9fb1ec374af8e1c511b8ddf10dc14ed0840701?context=explore)
does it work when you start it in docker compose?
I think we should use powsybl/java-dynawo ?